### ISSUE TITLE
Does not hide bar on Fedora 34, gnome 40 on wayland

### ISSUE THREAD

**Issue by udovdh** [mlutfy/hidetopbar#328]
Current state, with four days of uptime, is that the top bar is not hidden when I maximize thunderbird, firefox, etc.
How to fix this?

**Comment by tuxor1337** [comment]
I can't reproduce this. Would you please share more details about your setup?
* Operating system
* Version of GNOME
* Version of Hide Top Bar
* Hide Top Bar settings
* List of all GNOME extensions that are active in parallel

**Comment by udovdh** [comment]
Fedora 35
Gnome 41?
up to date
settings? Intellihide ONLY hide panel when a window takes the space
Desktop Icons NG, Hide Top Bar, Just Perfection, system-monitor, Vertical overview

**Comment by tuxor1337** [comment]
Okay, I can't reproduce this with the same Hide Top Bar settings, but with Fedora 34 and GNOME 40.6. However, I don't have any other GNOME extensions active in parallel. Can you check whether the issue still applies if there are no other GNOME extensions active in parallel?

**Comment by stecenator** [comment]
Extension disable and re-enable seems to help. 

Here is how I reproduce it:
1. Start laptop with dual monitor setup, main screen on external.
2. Switch monitor setup (via F7 on Lenovo) to "External only"
3. Top bar does not hide, covering top bars of maximized windows.

The monitor switching on X11 works fine
![HideTopBar](https://user-images.githubusercontent.com/26813596/147829213-611a6ddf-0630-48e2-a3af-6177fb564d05.png)

I'm on "intellihide" (all windows). 
I use `gTile` and `Sound Input & Output Device Chooser` in parallel, but the issue exists even when I disabled them.

I'm on up to date F35, nouveau.

**Comment by tuxor1337** [comment]
That sounds very much like a duplicate of #136

**Comment by tuxor1337** [comment]
@stecenator Please refer to #136, since your problem might be due to your multi-monitor setup.

@udovdh Reopening since you have a single display setup.
